pyteal/ast/abi/tuple.py

```diff
@@ -552,9 +559,50 @@ class User(abi.NamedTuple):
 
             my_user = User()
 
+    .. automethod:: __init_subclass__
     .. automethod:: __getattr__
     """
 
+    def __init_subclass__(cls) -> None:
+        """This method ensures one to construct only directly from `NamedTuple`,
```

Comment:
@ahangsu Minor and optional - It seems the wording can be clarified.  I don't feel strongly.

```suggestion
        """This method ensures one only constructs directly from `NamedTuple`,
```

Reply:
👍